src/_data/sites/uk-postoffice.yaml

```diff
@@ -0,0 +1,7 @@
+name: United Kingdom Post Office
+description: |-
+  Will not allow you to copy-paste your password into the text box (e.g. from a password manager) 
```

Comment:
Thanks for the contribution! 

Small nit, can you add a period at the end here and delete the trailing whitespace? It looks a bit off the way it is: https://dumb-password-rules-git-fork-m-davies-ukpostoffice-duffn.vercel.app/sites/united-kingdom-post-office/